www/resources/views/pages/testhistory.blade.php

```diff
@@ -52,14 +52,16 @@
                             <th><input type="checkbox" name="t[]" value="{{ $record->getTestId() }}" aria-label="Select this test" /></th>
                             <td class="url"><a href="/result/{{ $record->getTestId() }}/">{{ $record->getUrl() }}</a></td>
                             <td class="date">{{ date_format(date_create($record->getStartTime()), 'M d, Y g:i:s A e') }}</td>
-                            <td class="location">{{ $record->getLocation() }}</td>
+                            <td class="location"><div id="scrollable-wide-wrap">
```

Comment:
not sure why we need two div, also the scrollbar that way is overlapping with cell content, below solution works better please check
```
<td class="location" style="padding: unset;">
       <div style="overflow-x: auto;padding: 1em;">Test Scheduler 1182 - Chrome - 3GFast</div>
</td>
```